EMS-UI/src/app/service-definition/service-definition.component.ts

```diff
@@ -13,204 +13,32 @@ import { ToastrService } from 'ngx-toastr';
   templateUrl: './service-definition.component.html',
   styleUrls: ['./service-definition.component.css']
 })
-export class ServiceDefinitionComponent implements OnInit, OnChanges {
+export class ServiceDefinitionComponent implements OnChanges {
+
   @Input() selectedSupplier: number;
   @Input() selectedServiceDefinition: string;
-  @Input() selectedQueryType: string;
-
-  CdssUrl: any;
-  serviceDefinition: any;
-
-  tempSelectedServiceDefinition: any;
-  tempSelectedQueryType: any;
 
-  constructor(private serviceDefinitionService: ServiceDefinitionService, private toastr: ToastrService) {}
+  cdssSupplierId: number;
+  serviceDefinition: string;
+  tempSelectedServiceDefinitionId: string;
 
-  ngOnInit() {}
+  constructor(private serviceDefinitionService: ServiceDefinitionService) {}
 
   async ngOnChanges(changes: SimpleChanges) {
-    console.log(changes);
-
+    console.log("Changes Triggered");
```

Comment:
Is this just a debugging log? Should it be removed?

Reply:
oops, yes :)